tests/screenshot/apps/components/Item.js

```diff
@@ -5,7 +5,11 @@ const ItemTests = [
 	<Item>Hello Item</Item>,
 	<Item disabled>Hello Item</Item>,
 	<Item slotBeforeIcon="circlelarge" slotAfterIcon="circlelarge">Hello Item</Item>,
```

Comment:
`slotBeforeIcon` `slotAfterIcon` nee
Could you please add other props as well (inline, labelPosition, and so on)?